S/SPEX/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+using BinaryBuilder, Pkg
+
+name = "SPEX"
+version = v"1.1.5"
+
+# Collection of sources required to build SuiteSparse:GraphBLAS
+sources = [
+    GitSource("https://github.com/clouren/SPEX",
+        "9471682b072419063d73bc950949bf3458b187f9"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# Compile GraphBLAS
+cd ${WORKSPACE}/srcdir/SPEX
+atomic_patch -p1 ../werror.patch
+CFLAGS="${CFLAGS} -std=c99"
+cd ${WORKSPACE}/srcdir/SPEX/SPEX/SPEX_Util
+if [[ "${target}" == *mingw* ]]; then
+    make library -j${nproc} UNAME=Windows SO_OPTS="${SO_OPTS} -shared -L${libdir}"
+else
+    make library -j${nproc}
+fi
+cp ${WORKSPACE}/srcdir/SPEX/include/SPEX_Util.h ${includedir}
+cp ${WORKSPACE}/srcdir/SPEX/lib/libspexutil* ${libdir}
+cd ${WORKSPACE}/srcdir/SPEX/SPEX/SPEX_Left_LU
+if [[ "${target}" == *mingw* ]]; then
+    make library UNAME=Windows SO_OPTS="${SO_OPTS} -shared -L${libdir}"
+else
+    make library -j${nproc}
+fi
+cp ${WORKSPACE}/srcdir/SPEX/include/SPEX_Left_LU.h ${includedir}
+cp ${WORKSPACE}/srcdir/SPEX/lib/libspexleftlu* ${libdir}
+install_license ${WORKSPACE}/srcdir/SPEX/SPEX/LICENSE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(;experimental=true)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libspexutil", :libspexutil),
+    LibraryProduct("libspexleftlu", :libspexleftlu)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency("SuiteSparse_jll"),
+    Dependency("GMP_jll"),
+    Dependency("MPFR_jll")
```

Comment:
#3024

Reply:
What counts as "old" Julia versions? Pre 1.6? 

So should I depend on that specific version in all cases?

Reply:
Yes, or more simply use the version in Julia v1.6 (that's what we target anyway): https://github.com/JuliaLang/julia/blob/57cd92c5d89cd3e2b66ac4d632e81c1f06562da4/stdlib/MPFR_jll/Project.toml#L3 and https://github.com/JuliaLang/julia/blob/release-1.6/stdlib/GMP_jll/Project.toml#L3

Reply:
Correct versions I think